cli/tests/testdata/lint/without_config/malformed.js

```diff
@@ -1,4 +1,4 @@
 // deno-fmt-ignore-file
 
 // intentionally malformed file
-export class A {
```

Comment:
deno_lint works now at analyzing files like this so I changed it to something it will fail analyzing.